### ISSUE TITLE
Warning: Unexpected procedure property volatility returned from Snowflake

### ISSUE THREAD

**Issue by nicolamarangoni** [Snowflake-Labs/terraform-provider-snowflake#2536]
### Terraform CLI and Provider Versions

Snowflake Provider 0.86

### Terraform Configuration

```terraform
resource "snowflake_procedure" "my_procedure" {
  name        = "MY_PROCEDURE"
  database    = "MY_DATABASE"
  schema      = "PUBLIC"
  language    = "JAVASCRIPT"
  return_type = "VARCHAR(16777216)"
  execute_as  = "CALLER"
  statement   = <<EOT
....
....
EOT
}
```

### Expected Behavior

Plan/Apply without warning

### Actual Behavior

Plan triggers the following warning:

```
│ Warning: Unexpected procedure property returned from Snowflake
│ 
│   with snowflake_procedure.my_procedure,
│   on snowflake_procedures.tf line 1, in resource "snowflake_procedure" "my_procedure":
│    1: resource "snowflake_procedure" "my_procedure" {
│ 
│ Unexpected procedure property volatility returned from Snowflake
│ 
│ (and 5 more similar warnings elsewhere)
```

### Steps to Reproduce

1. `terraform plan`

### How much impact is this issue causing?

Low

**Comment by sfc-gh-asawicki** [comment]
Hey @nicolamarangoni. Thanks for reaching out to us.

This is a harmless warning. We will demote it to INFO in one of the next versions.

**Comment by nicolamarangoni** [comment]
Hi @sfc-gh-asawicki , many thanks for the info!

**Comment by sfc-gh-asawicki** [comment]
Hey @nicolamarangoni. We have released the fix as part of [v0.87.0](https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases/tag/v0.87.0) release. Please follow the [migration guide](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/v0.87.0/MIGRATION_GUIDE.md#v0860--v0870) during the update. Please confirm that the issue is resolved in the newest version. Thanks!

**Comment by nicolamarangoni** [comment]
Hello, I don't see the warning anymore! Thanks!
